my_game.py

```diff
@@ -21,6 +21,11 @@
 PLAYER_THRUST_KEY = arcade.key.UP
 PLAYER_FIRE_KEY = arcade.key.SPACE
 
+TEXTURES = [
```

Comment:
Add comment (change name?) so we know what these are for.

Reply:
I changed it ok? I did! Yeah! To something much better! Mhm! Mhm! Yeah!